pyo3-derive-backend/src/pyclass.rs

```diff
@@ -321,6 +324,18 @@ fn impl_class(
         quote! { None }
     };
 
+    // Enforce at compile time that PyGCProtocol
+    let gc_impl = if has_gc {
+        quote! {
+            use pyo3::class;
+
+            fn _assert_implements_protocol<T: pyo3::class::PyGCProtocol>() {};
```

Comment:
This `:` looks redundant

Reply:
Which one? Isn't it `fn func_name<GenericName: TraitBound>() {};`?

Reply:
Sorry typo of `;`